app/_locales/en/messages.json

```diff
@@ -1741,6 +1741,20 @@
     "message": "Your $1 will be added to your account once this transaction has processed.",
     "description": "This message communicates the token that is being transferred. It is shown on the awaiting swap screen. The $1 will be a token symbol."
   },
+  "swapPriceDifference": {
+    "message": "You are about to swap $1 $2 (~$3) for $4 $5 (~$6).",
+    "description": "This message represents the price slippage for the swap.  $1 and $4 are a number (ex: 2.89), $2 and $5 are symbols (ex: ETH), and $3 and $6 are native currency amounts."
```

Comment:
Native currency amounts? We use the term "native currency" throughout the codebase to describe the currency native to the current _network_ (i.e. ETH). I'm guessing that $3 and $6 are the _fiat_ currency.

Reply:
Thank you!  Updated!